src/main/java/baritone/process/BuilderProcess.java

```diff
@@ -474,6 +504,10 @@ public int lengthZ() {
                     }
                 }
             }
+            if(ignoringBuildIgnoreExisting){
```

Comment:
bruh the fucking spaces dude

Reply:
I used notepad for this, and it shoved tabs in there instead of whitespaces. Sorry.